lib/mutant/reporter/cli/printer.rb

```diff
@@ -3,10 +3,34 @@ class Reporter
     class CLI
       # CLI runner status printer base class
       class Printer
-        include AbstractType, Delegator, Adamantium::Flat, Concord.new(:output, :object), Procto.call(:run)
+        include AbstractType,
+                Adamantium::Flat,
+                Concord.new(:output, :object),
+                Procto.call(:run)
 
         private_class_method :new
 
+        # Create delegators to object
+        #
+        # @return [undefined]
+        def self.delegate(*names)
```

Comment:
This change is about to in-line the `Mutant::Delegator` with its only call sides. It was a module as I thought delegation to private `#object` would be a more common pattern. But it only turned out to exist in the `Mutant::Reporter::CLI::Printer*` namespace so I decided to remove `Mutant::Delegator` with this PR.